pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -631,19 +627,6 @@ func validateInterfaceNameUnique(field *k8sfield.Path, networkInterfaceMap map[s
 	return causes
 }
 
-func validateNetworkInterfaceMultiqueue(field *k8sfield.Path, vifMQ *bool, isMultiQueueAllowed bool) (causes []metav1.StatusCause) {
-	if vifMQ != nil && *vifMQ && !isMultiQueueAllowed {
-
-		causes = append(causes, metav1.StatusCause{
-			Type:    metav1.CauseTypeFieldValueInvalid,
-			Message: "virtio-net multiqueue request, but there are no virtio interfaces defined",
-			Field:   field.Child("domain", "devices", "networkInterfaceMultiqueue").String(),
-		})
-
-	}
-	return causes
-}
```

Comment:
why are you removing the vmi admitting code ? Wouldn't it make sense to reject a request for multi-queue **if** an the corresponding VM features a non-virtio interface ?

Reply:
This what concluded in the long discussion here
https://github.com/kubevirt/kubevirt/issues/7671

That we relax it, it will be best effort
and present queueCount

I think it do belong to the mentioned PR instead please
https://github.com/kubevirt/kubevirt/pull/8071

Reply:
I think relaxing it is in fact making it less deterministic, which is only making it more confusing. 

But if you've agreed on it ... I guess I missed the opportunity to voice my concerns.

Reply:
Relaxing means that we cannot maintain endless conditions of permutations.
The complexity is due to an API which supported defaults and the notion of parameters that apply on multiple interfaces.

After 2 or 3 incidents of bugs due to this matrix of permutations, we managed to convinced some of the maintainers that this is not maintainable. Thus, relaxing the strictness in favor of reflecting the actual state (number on queues).

My lesson out of this is basically to never try and abstract an API by introducing defaults and template parameters instead of individual ones (e.g. queue count per VMI and not per interface).